server/src/test/java/io/crate/metadata/settings/SessionSettingsTest.java

```diff
@@ -36,7 +36,7 @@ public class SessionSettingsTest {
 
     @Test
     public void testSessionSettingsStreaming() throws IOException {
-        SessionSettings s1 = new SessionSettings("user", SearchPath.createSearchPathFrom("crate"), true, false, 20);
+        SessionSettings s1 = new SessionSettings("user", SearchPath.createSearchPathFrom("crate"), true, false, 20, false);
```

Comment:
`false` is the default value, so streaming the correct value is only ensured when using `true`.
```suggestion
        SessionSettings s1 = new SessionSettings("user", SearchPath.createSearchPathFrom("crate"), true, false, 20, true);
```